app/models/shift_commerce/application_record.rb

```diff
@@ -0,0 +1,5 @@
+module Shiftcommerce
+  class ApplicationRecord < ActiveRecord::Base
+    self.abstract_class = true
+  end
+end
```

Comment:
Remove this please – front-end websites shouldn't need ActiveRecord as models are loaded via the API.